src/alttxt/enums.py

```diff
@@ -68,9 +68,9 @@ class SortBy(Listable):
     Strings need to be kept up-to-date with any changes
     to the MultiNet implementation's export format.
     """
-    DEGREE = "Degree"
-    SIZE = "Size"
-    DEVIATION = "Deviation"
+    DEGREE = "degree"
+    SIZE = "size"
+    DEVIATION = "deviation"
```

Comment:
Are these used any more? Maybe the can be deleted? I don't see them in the multinet-api repo

Reply:
I believe sort by is used in GrammarModel and has been accessed as grammar["sortBy"]... so thus I am keeping it.